darshan_logs/imbalanced_io/README.md

```diff
@@ -1 +1,7 @@
 This example is an anonymized Darshan log that represents a heavily imbalanced I/O workload. In particular, one can observe a balanced I/O workload to file record 15708535418621378501 at the MPI-IO layer, but at the POSIX layer there is a clear imbalance (see SLOWEST_RANK_BYTES and FASTEST_RANK_BYTES counters). The Lustre module stripe info makes clear this issue is due to this file being striped across only a single file server -- in this case MPI-IO picks a single writer process for POSIX I/O, leading to a large imbalance.
+
+This log is also an example of a log file with partial module data. The job
+opened more than 1024 files on rank 0, causing it to run into the default
+limit on the number of file records that Darshan's POSIX module will track
+on each process.  This can be observed by runnig `darshan-parser` with and
```

Comment:
`runnig` -> `running`

Apart from that, seems "ok" to me.

I agree it doesn't hurt to have the extra examples, I was mostly looking for clarity on the situation--I was under the impression we had *no* such files based on the discussions I had seen, now I'm less convinced since I just checked a random one and found it was partial.

Reply:
(in other words, when submitting new log files, confirm that the property does or does not exist in all other log files in the logs repo, to give the reveiwers a clear picture, even if the decision is to merge anyway)

Reply:
Thanks for the catch, pushed squashed commit with fix in place just now.

That's good advice for best practice when submitting new logs.  I'll capture that in the top level readme separately.